src/DifferenceEquations.jl

```diff
@@ -0,0 +1,17 @@
+module DifferenceEquations
+
+using SciMLBase
+using Distributions
+
+import SciMLBase: SciMLProblem
```

Comment:
```suggestion
using SciMLBase: SciMLProblem
```
Do you need anything else from SciMLBase? If not, you could remove `using SciMLBase` above.

Reply:
As of right now, I don't think so. Might add in some other interface methods but I suppose I can do away with the blanket `using`.